core/esmf-aspect-model-document-generators/src/test/java/org/eclipse/esmf/aspectmodel/generator/sql/databricks/AspectModelDatabricksDenormalizedSqlVisitorTest.java

```diff
@@ -106,7 +106,7 @@ CREATE TABLE IF NOT EXISTS aspect_with_complex_collection_enum (
    void testAspectWithComplexEntityCollectionEnum() {
       assertThat( sql( TestAspect.ASPECT_WITH_COMPLEX_ENTITY_COLLECTION_ENUM ) ).isEqualTo( """
             CREATE TABLE IF NOT EXISTS aspect_with_complex_entity_collection_enum (
-              my_property_one__entity_property_one ARRAY<STRUCT<entity_property_two: STRING NOT NULL>> NOT NULL
+              entity_property_two STRING NOT NULL
```

Comment:
This change is not correct; the nested structure of `myPropertyOne`/`entityPropertyOne` still needs to be present, but flattened by name concatenation. I.e., this line should read:
`my_property_one__entity_property_one__entity_property_two STRING NOT NULL`